modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -10,14 +10,19 @@ Review the following limitations on support for local registries:
 [IMPORTANT]
 You cannot use local registries during the installation of OpenShift operator. However, you can use local registries with OpenShift RTF to store Mule Docker images and install and manage Runtime Fabric for running Mule apps. 
 
-* You must use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or later. Note that `rtfctl` installs Runtime Fabric agent to the latest version by default.
+* You can use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or later. Note that `rtfctl` installs Runtime Fabric agent to the latest version by default.
+* You can use Helm to install Rutime Fabric and configure your local registry.
```

Comment:
Is this a limitation?

Reply:
Is not a limitation. Users can use Helm or rtfctl to install Runtime Fabric